tests/sparseml/core/logger/utils/test_frequency_manager.py

```diff
@@ -137,3 +137,29 @@ def test_log_ready(
     )
 
     assert actual == expected
+
+
+def _test_cases__validate_log_frequency():
+    # "log_frequency, frequency_type, expectation"
+
+    return [
```

Comment:
why not just include below with the pytest call?

Reply:
No reason in particular, will move it below